### DISCUSSION TITLE
[RELEASE] Dein Sturz

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#174] [Map Releases]
This is my second Map called Dein Sturz this was inspired by the Call of Duty: Black Ops 2 Zombies Map "Die Rise"

It features a pretty view onto the Streets.
It's overall a pretty _decently_ sized Map.

> Pain deluxe before aquiring the PSP _"Making maps crossplatform sucks and screw psp support my maps will look good and there's nothing you can do about it I only want 3DS support (:<<<<<"_
> 
> Pain deluxe after aquiring the PSP _"Why did I say that..."_

Note this from above was due to the first Read me I made when I made that map.

Also do not expect a great amount of FPS on the PSP!

### [DOWNLOAD](https://github.com/nzp-team/nzportable/files/10609714/Dein.Sturz.zip)

**Comment by MotoLegacy** [comment]
Hey there! This map isn't eligible for the "Meets Install Standard" label, you can check out the requirements and why it's useful on the post for it: https://github.com/nzp-team/nzportable/discussions/159

**Reply by Paindeluxe** [reply]
Fixed! Sorry 'bout that.
